TombEngine/Renderer/RendererDraw.cpp

```diff
@@ -3254,69 +3291,87 @@ namespace TEN::Renderer
 	{
 		if (rendererPass == RendererPass::CollectTransparentFaces)
 		{
-			for (auto& bucket : mesh->Buckets)
+			for (int animated = 0; animated < 2; animated++)
```

Comment:
I don't fully understand why this for loop is needed here? It seems that `animated` is not used in this code part.

Reply:
Done